universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -166,6 +192,21 @@ private Question applicantIceCream(QuestionEnum ignore) {
     return maybeSave(definition);
   }
 
+  // File upload
+  private Question applicantFile(QuestionEnum ignore) {
```

Comment:
(blocking) these arguments are for a name question. can you change these to be for a file upload question?

Reply:
Resolved and made other question attributes more consistent